packages/amplify-provider-awscloudformation/lib/transform-graphql-schema.js

```diff
@@ -177,14 +188,15 @@ async function transformerVersionCheck(context, resourceDir, cloudBackendDirecto
       if (!response.transformerConfig) {
         process.exit(0);
       }
+      transformerVersion += 1;
```

Comment:
This should be the current version of the transformer. If the customer goes and edits the version to 25 or what ever value we should not be making it 26. There is also the possiblilty of team workflow where one developer is in one version and the other is in a different version. When a older version is used, we should warn them as well as this could break things

Reply:
Let's say a customer is in version 25 and the current version is 27. If they acknowledge the prompt wouldn't they potentially miss a warning message for version 26?